components/hydrator/internal/subject/cert_subject.go

```diff
@@ -40,7 +40,8 @@ func NewProcessor(ctx context.Context, certSubjectMappingCache certsubjectmappin
 
 	for _, m := range mappings {
 		if err := m.Validate(); err != nil {
-			return nil, err
+			// in case the certificate subject mapping is invalid, only log an error and continue so the hydrator component can still start
```

Comment:
minor: the go-doc of this func can be adapted as well:

`If the subject-to-consumer mapping is invalid, an error is returned.` -> `If the subject-to-consumer mapping is invalid, an error is logged.`